packages/backup-restore-operator/charts/templates/backup.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: backups.resources.cattle.io
+spec:
+  group: resources.cattle.io
+  names:
+    kind: Backup
+    plural: backups
```

Comment:
```suggestion
    listKind: BackupList
    plural: backups
    singular: backup
```

Reply:
All CRD manifests were generated by the wrangler openapi schema generator, is there a reason this should be changed?

Reply:
Oh, I wasn't aware that these CRD definitions were auto-generated. I've seen these fields in a couple of the Prometheus Operator CRDs, wasn't sure if they would be required here. 

i.e. with this CRD installed, can you do `kubectl get backup` or do you need to do `kubectl get backups`?

Reply:
I can do both right now

Reply:
No need to make these changes then, will resolve 👍